tck/src/main/java/org/eclipse/microprofile/config/tck/ConfigProviderTest.java

```diff
@@ -19,6 +19,7 @@
  */
 package org.eclipse.microprofile.config.tck;
 
+import java.io.*;
```

Comment:
Hm, 5 imports instead of 1 - I can feel the pain before comitting :/ :

```
import java.io.ByteArrayInputStream;
import java.io.ByteArrayOutputStream;
import java.io.IOException;
import java.io.ObjectInputStream;
import java.io.ObjectOutputStream;
```